bitnami/drupal/templates/externaldb-secrets.yaml

```diff
@@ -6,5 +6,5 @@ metadata:
   labels: {{- include "common.labels.standard" . | nindent 4 }}
 type: Opaque
 data:
-  db-password: {{ default "" .Values.externalDatabase.password | b64enc | quote }}
+  mariadb-password: {{ default "" .Values.externalDatabase.password | b64enc | quote }}
```

Comment:
Wouldn't this change break compatibility with mariadb helm chart (which is used when `mariadb.enabled: true`) compatibility which requires `mariadb-password`: 
https://github.com/bitnami/charts/blob/051c5bebebbd6975e0261e42cf0ec9be24f8100b/bitnami/mariadb/templates/secrets.yaml#L28

Reply:
I am not sure aI get your point, could you please elaborate on it?